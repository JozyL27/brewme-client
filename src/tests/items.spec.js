import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Beer from '../components/searchBeers/items'
import { testInput } from './test-helpers'



describe(`items component`, () => {
    it(`items component renders without crashing`, () => {
        const div = document.createElement('div')
        ReactDOM.render(<Beer userId={5} brewskis={testInput} />, div)
        ReactDOM.unmountComponentAtNode(div)
    })
})