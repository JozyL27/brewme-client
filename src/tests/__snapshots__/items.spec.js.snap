// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`items component renders the UI as expected 1`] = `
Array [
  <li
    className="searchLi"
  >
     
    beer one
    <div
      className="searchAbv"
    >
      ABV: 
      4
    </div>
    <p
      className="searchDescript"
    >
      im a beer
    </p>
    <button
      onClick={[Function]}
    >
      Add to My Beers
    </button>
  </li>,
  <li
    className="searchLi"
  >
     
    beer two
    <div
      className="searchAbv"
    >
      ABV: 
      5
    </div>
    <p
      className="searchDescript"
    >
      im a beer too
    </p>
    <button
      onClick={[Function]}
    >
      Add to My Beers
    </button>
  </li>,
]
`;
